pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -15,18 +15,27 @@
 This module contains the next generation successor to default qubit
 """
 
+from functools import partial
 from typing import Union, Callable, Tuple, Optional, Sequence
+import concurrent.futures
+import multiprocessing
+import cloudpickle
 
 import pennylane.numpy as np
 from pennylane.tape import QuantumTape, QuantumScript
-from pennylane.typing import Result, ResultBatch
+from pennylane.typing import Result, ResultBatch, ArrayBox
```

Comment:
I would recommend `ArrayBox` directly from `autograd.numpy.numpy_boxes` instead of from typing.